README.md

```diff
@@ -372,6 +373,9 @@ Nginx API for php
 * [ngx_header_get](#ngx_header_get)
 * [ngx_header_gets](#ngx_header_gets)
 * [ngx_redirect](#ngx_redirect)
+* [ngx_cookie_gets](#ngx_cookie_gets)
```

Comment:
I think will be `better ngx_cokie_get_all()`

Reply:
Update later~